web/src/pages/Route/locales/en-US.ts

```diff
@@ -48,7 +48,7 @@ export default {
 
   'page.route.form.itemRulesRequiredMessage.parameterName':
     'Only letters and Numbers are supported, and can only begin with letters',
-  'page.route.value': 'Value',
+  'page.route.value': 'Parameter Value',
```

Comment:
Why to change this, `Value` is fine IMO

Reply:
Yes, without the Parameter is also ok here. @iamayushdas 
Parameter is added only to be consistent with the Chinese version.

![image](https://user-images.githubusercontent.com/75366457/125195829-0f00b180-e28a-11eb-990e-22ccee1ad710.png)